pmd-core/src/main/java/net/sourceforge/pmd/lang/document/FragmentedTextDocument.java

```diff
@@ -46,46 +46,53 @@ public LanguageVersion getLanguageVersion() {
     }
 
     @Override
-    protected int localOffsetTransform(int outOffset, boolean inclusive) {
-        // caching the last accessed fragment instead of doing
```

Comment:
Same comment. I don't think splitting the implementation achieves anything, and you removed all comments.